### DISCUSSION TITLE
Nvidia-Omniverse模擬平臺

### DISCUSSION THREAD

**Discussion by boson13579** [1101-GS4719-C/Python-Community-GS4719-C#1425] [General]
我看了一篇有關於Nvidia和3D繪圖的文章。我覺得這個技術真的很厲害！文章裡提到，用3D繪圖軟體模擬拿水倒入玻璃杯的動作需要很多的細節和技術。而且，Nvidia的執行長還用虛擬技術在演講中展示自己在廚房講話，真的很神奇！未來，這種3D繪圖和物理模擬技術還可以用來建造虛擬世界，讓人們在其中互動交流。我對這個新技術感到很興奮，希望有機會可以體驗一下！
新聞來源:
https://www.ithome.com.tw/news/148177

**Comment by brian431** [comment]
此篇報導整理地十分完整。我個人也是非常期待這項技術的發展，不過這些元宇宙相關的技術要真正融入大眾的生活中感覺還需要一段時間。

**Comment by TW-Maverick** [comment]
最近有看到類似的技術，利用激光雷達和 SLAM（同時定位和映射），允許機器人和其他自動駕駛車輛同時構建地圖並在該地圖上定位自己，優勢是可以到人類無法到達的地方探索，Ex.其他行星、很窄的下水道🤣。

我覺得3D視覺技術很熱門，可以歸功於自駕車、AR/VR，希望未來台灣會有「Cruise」提供服務！
